pinecone/sparse/splade/splade-vector-generation.ipynb

```diff
@@ -281,128 +297,114 @@
     },
```

Comment:
We should probably give a one-line explanation what Splade is, what is it meant for and how it achieves that.

Reply:
We have a link to an explanation